spring-test/src/test/java/org/springframework/test/context/TestExecutionListenersTests.java

```diff
@@ -126,13 +126,13 @@ void defaultListenersMergedWithCustomListenerInserted() {
 
 	@Test
 	void nonInheritedDefaultListeners() {
-		assertRegisteredListeners(NonInheritedDefaultListenersTestCase.class, asList(QuuxTestExecutionListener.class));
+		assertRegisteredListeners(NonInheritedDefaultListenersTestCase.class, List.of(QuuxTestExecutionListener.class));
```

Comment:
`asList()` is used consistently within this test class. Switching to `List.of()` in a few places therefore doesn't add value. We should use one or the other but not mix both.

So let's just leave it as-is.

In light of that, please revert the changes in this class.

Reply:
Reverted [60b8775](https://github.com/spring-projects/spring-framework/pull/29413/commits/60b8775a94b890e293fae083ae540adf77b34c0a)